src/plugins/github/demoData/regenerate.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eu
+
+main() {
+  if [ -z "${GITHUB_TOKEN:-}" ]; then
+    printf >&2 'Please set the GITHUB_TOKEN environment variable\n'
+    printf >&2 'to a 40-character hex string API token from GitHub.\n'
+    return 1
+  fi
+  yarn backend >/dev/null;
```

Comment:
(If only `chronic(1)` were part of `coreutils` and not [`moreutils`]… but this is superseded by general comment about this choice, anyway.)

[`moreutils`]: https://joeyh.name/code/moreutils/